### ISSUE TITLE
CPU Cycles setting names changed?

### ISSUE THREAD

**Issue by ToniBC** [libretro/dosbox-svn#26]
Hello, I tell you this because I think the names to adjust the CPU Cycles in the options are changed.

![CPU Cycles Names](https://user-images.githubusercontent.com/45365478/68982673-3fb7eb80-0808-11ea-9426-3e23afb6d72b.JPG)

The value would have to be the largest number and the multiplier from 0 to 9, I think it's the most logical. It's a small detail and it's just changing the name, but I think it's wrong as seen in the photo.

Thank you.

**Comment by realnc** [comment]
Not sure what you mean.

**Comment by realnc** [comment]
Oh, never mind. I get it. "Value" and "multiplier" should be swapped in the option names.

**Comment by ToniBC** [comment]
Exactly, the names of value and multiplier would have to be exchanged so that it is well understood. It is a minor detail, the result is the same, but I found some who did not understand and get confused.

![CPU Cycles Names ok](https://user-images.githubusercontent.com/45365478/68991483-a4526500-085f-11ea-9799-1853c9765d68.jpg)
